netket/hilbert/continuous_hilbert.py

```diff
@@ -0,0 +1,67 @@
+from typing import Tuple
+
+from .abstract_hilbert import AbstractHilbert
+
+class Particles(AbstractHilbert):
+    """Class for continuous Hilbert space.
```

Comment:
This is not for continouus hilbert space.
It's rather for particles in a continous hilbert space.

again, mayube one day we'll have a proper base class for abstracthilberts (and particles ain't it).

Reply:
Please change this class docstring to something more proper

Reply:
This class (for now) it's more the hilbert space for N particles in a n-dimensional box.